app/controllers/annotation_reception_controller.rb

```diff
@@ -1,6 +1,29 @@
 class AnnotationReceptionController < ApplicationController
   skip_before_action :verify_authenticity_token, only: [:update]
 
-  def update; end
+  def update
+    annotation_reception = AnnotationReception.find_by!(uuid: params[:uuid])
+    annotations_col = get_result_from_json_body
 
+    annotation_reception.process_annotation!(annotations_col)
+
+    respond_to do |format|
+      format.any {head :no_content}
+    end
```

Comment:
`format.any`を使うのであれば、単に`head :no_content`すれば良さそうです。
```suggestion
    head :no_content
```